### PR TITLE
 Multiple redis instances

### PR DESCRIPTION
**Pull request by thiagocaiubi**
This PR depends on #4

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -50,14 +50,14 @@ func readConfiguration(fileName string) configuration {
...
-		log.Fatalf("Error connecting to Redis: %s", err)
+		log.Printf("Error connecting to Redis: %s", err)
 		return
 	}
 	defer connection.Close()
 
 	reply, err := connection.Do("INFO")
 	if err != nil {
-		log.Fatalf("Error doing INFO command: %s", err)
+		log.Printf("Error doing INFO command: %s", err)
```
Comment (mateusduboli) [inline main.go]: Rather see this separated in another PR

Comment (thiagocaiubi) [inline main.go]: :+1:

Comment (thiagocaiubi) [inline main.go]: Done. Please, see #6

**Comment by mateusduboli** [comment]
LGTM

**Pull request merged**